catalog/transformer/catalog-transformer-overview/src/test/java/ddf/catalog/transformer/OverviewMetacardTransformerTest.java

```diff
@@ -0,0 +1,143 @@
+
+        final BufferedImage originalImage = getImage(getImageBytes());
+        final BufferedImage overviewImage = getImage(content.getByteArray());
+
+        assertThat(overviewImage.getWidth(), greaterThan(originalImage.getWidth()));
+        assertThat(overviewImage.getHeight(), greaterThan(originalImage.getHeight()));
+    }
+
+    private BufferedImage getImage(final byte[] imageBytes) throws IOException {
+        try (final ByteArrayInputStream inputStream = new ByteArrayInputStream(imageBytes)) {
+            final ImageInputStream imageInputStream = ImageIO.createImageInputStream(inputStream);
+            return ImageIO.read(imageInputStream);
+        }
+    }
+
+    private byte[] getImageBytes() throws IOException {
+        try (final InputStream inputStream = getClass().getClassLoader()
+                .getResourceAsStream("test.jpg")) {
+            return IOUtils.toByteArray(inputStream);
+        }
+    }
+
+    private BinaryContent transform(final Metacard metacard,
+            final Map<String, Serializable> arguments)
+            throws CatalogTransformerException, IOException {
+        final BinaryContent content = transformer.transform(metacard, arguments);
+        verifyImageIsPng(content.getByteArray());
+        return content;
+    }
+
+    private void verifyImageIsPng(final byte[] imageBytes) {
+        // Check the PNG header bytes.
+        assertThat(imageBytes[0], is((byte) 0x89));
+        assertThat(imageBytes[1], is((byte) 0x50));
+        assertThat(imageBytes[2], is((byte) 0x4E));
+        assertThat(imageBytes[3], is((byte) 0x47));
+        assertThat(imageBytes[4], is((byte) 0x0D));
+        assertThat(imageBytes[5], is((byte) 0x0A));
+        assertThat(imageBytes[6], is((byte) 0x1A));
+        assertThat(imageBytes[7], is((byte) 0x0A));
+    }
+
+    @Test(expected = CatalogTransformerException.class)
+    public void testNoOverviewAttribute() throws CatalogTransformerException, IOException {
+        transform(new MetacardImpl(), new HashMap<>());
+    }
+
+    @Test(expected = CatalogTransformerException.class)
+    public void testNoLocation() throws Exception {
+        final MetacardImpl metacard = getMetacardWithThumbnail();
+
+        final Map<String, Serializable> arguments = new HashMap<>();
+        arguments.put("align", "true");
+        transform(metacard, arguments);
+    }
+
+    @Test
+    public void testInvalidWkt() throws Exception {
+        final MetacardImpl metacard = getMetacardWithThumbnail();
+        metacard.setLocation("INVALID WKT");
+
+        final Map<String, Serializable> arguments = new HashMap<>();
+        arguments.put("align", "true");
+        try {
+            transform(metacard, arguments);
+            fail("Should have thrown a CatalogTransformerException.");
+        } catch (CatalogTransformerException e) {
+            assertThat(e.getCause(), instanceOf(ParseException.class));
+        }
```

Comment:
Refactored to use [`ExpectedException`](http://junit.org/junit4/javadoc/latest/org/junit/rules/ExpectedException.html).